packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# 47.17.1 - January 30, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-apex
+
+- Update Apex Language Server error message to include we support Java 8 and Java 11 ([PR #1900](https://github.com/forcedotcom/salesforcedx-vscode/pull/1900))
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Set cursor on the first argument when inserting method from code completion ([PR #1913](https://github.com/forcedotcom/salesforcedx-vscode/pull/1913))
```

Comment:
I'm adding images and gifs to highlight some of the changes done. You'll have to "manually" update the image url if you want to preview them since I have the link using `develop` branch.